MAUI/Cartesian-Charts/Getting-Started.md

```diff
@@ -16,14 +16,29 @@ To get start quickly with our .NET MAUI Cartesian Chart, you can check the below
 
 {% youtube "https://www.youtube.com/watch?v=o616GkzdPJk&t=7s" %}
 
-## Creating an application with .NET MAUI chart
+## Prerequisites
 
-1. Create a new .NET MAUI application in Visual Studio.
-2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfCartesianChart to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.Charts and then install it.
-3. To initialize the control, import the Chart namespace.
-4. Initialize [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
+Before starting, ensure the following are set up:
 
-{% tabs %} 
+1. .NET 7 SDK or later is installed.
```

Comment:
[.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)

Reply:
Updated the link